### ISSUE TITLE
Typo on Autoscalar

### ISSUE THREAD

**Issue by paulbellamy** [pulumi/pulumi-gcp#313]
Probably meant to be 'Autoscaler'. :)

**Comment by stack72** [comment]
Hi @paulbellamy 

My apologies for this - I will get this fixed and up to date today and make a release


Note to myself:

```
			"google_compute_autoscaler": {
				Tok: gcpResource(gcpCompute, "Autoscalar"),
				Docs: &tfbridge.DocInfo{
					Source: "compute_autoscaler.html.markdown",
				},
			},
```

This resource needs renamed

**Comment by paulbellamy** [comment]
All good! :) Overall pulumi is pretty nice so far, just thought it was funny.

**Comment by stack72** [comment]
yeah this is 100% my typo 🥇 LOL

**Comment by stack72** [comment]
Hi @paulbellamy this is fixed via #314 and will be released in a couple of hours
